### ISSUE TITLE
[auto_thumb.php] qui plante si CPU trop faible

### ISSUE THREAD

**Issue by Nh3xus** [broncowdd/BoZoN#148]
Sur un Raspberry Pi, si j'upload un dossier contenant beaucoup d'images, le script auto_thumb.php est lent à s'exécuter et Apache fini par m'informer que le script a dépassé le temps limite d'exécution autorisé pour les scripts :

PHP Fatal error:  Maximum execution time of 600 seconds exceeded in /var/www/test/core/auto_thumb.php

J'ai rajouté moi même la valeur de 600 secondes de timeout au début du fichier auto_thumb mais ça ne change rien.

Curieusement, les vignettes finissent tant bien que mal par être générées.

Merci.

**Comment by broncowdd** [comment]
Oui, le RPI peut s'avérer un peu juste en cas de génération massive des 
miniatures... Pourtant, le lazy load devrait résoudre le problème. Je le 
note et je vois ;-) Merci !
